test/Table_Tests/src/Util.enso

```diff
@@ -50,15 +50,17 @@ type Test_Failure_Error
     to_display_text self = "Test failure error: "+self.message
 
 ## PRIVATE
-table_should_equal_impl actual expected loc =
-    case expected of
+table_should_equal_impl actual_table expected_table loc ignore_order =
+    case expected_table of
         _ : Table ->
+            actual = if ignore_order then actual_table.sort else actual_table
+            expected = if ignore_order then expected_table.sort else expected_table
```

Comment:
`sort` with no arguments sorts the tables on the first column.

I can imagine comparing tables
| A | B |
|---|---|
| a | 1 |
| a | 2 |
| b | 3 |

and

| A | B |
|---|---|
| a | 2 |
| a | 1 |
| b | 3 |

These two tables should be "equal ignoring order", but the current setup will find them not-equal because they are already sorted by the first column.

Reply:
I think we nee to:
- consider sorting over all columns? So that the ordering can 'stabilize' regardless of which columns actually influence the order.
- consider comparing vectors of rows using something like existing `should_equal_ignoring_order` or something implemented in a similar way?

Reply:
Probably good to add a test with the tables as shown above to the `Util_Spec`.

Reply:
Done